src/todo/components/TodoList.tsx

```diff
@@ -1,16 +1,22 @@
-import React, { useState } from "react";
+import React, { useState, useEffect, useMemo } from "react";
 import '../../index.css';
 import TodoItem from "./TodoItem";
-import { DEFAULT_TODO_ITEM } from "./const";
-import { createNewTodo } from "./const"
+import { DEFAULT_TODO_ITEM, createNewTodo } from "./const";
+import { getList } from "./api";
+
 
 const ToDo = () => {
-    const [name, setName] = useState('');
+    
     const [todos, setTodos] = useState(DEFAULT_TODO_ITEM);
     const [title, setTitle] = useState('')
     const [description, setDescription] = useState('')
 
-    const handleAddTask = e => {
+
+    useEffect(() => {
```

Comment:
IMHO it's not good solution
better way to add/delete/update items in the appropriate handlers

Reply:
as for me, better order code by the rules

first - hooks: useState, useMemo, useEffect
then handlers
then variables for render 
and at the end - render functions

so code will be easy readable

Reply:
Do you mean just swap places?

Reply:
yes

Reply:
Can I leave "memoizedTodoItem" in its place? I moved the "useEffect" hooks to the top. Just "memoizedTodoItem it's already render?

Reply:
it's up to you